test/functional/wallet_watchonly.py

```diff
@@ -94,6 +98,14 @@ def run_test(self):
         assert_equal("psbt" in result, True)
         assert_raises_rpc_error(-4, "Insufficient funds", wo_wallet.walletcreatefundedpsbt, inputs, outputs, 0, no_wo_options)
 
+        self.log.info('Testing sendmany watch-only')
```

Comment:
you should test that wallet balances aren't changing here(accidentally committing the transaction internally or something) and that it would enter the mempool properly